release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -80,6 +80,11 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-openshift-cli"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-16-oc-adm-upgrade-status-tp"]
+==== Introducing the `oc adm upgrade status` command (Technology Preview)
+
+Previously, the `oc adm upgrade` command provided limited information about the status of a cluster update. With this release, there is now a technology preview `oc adm upgrade status` command in addition to `oc adm upgrade`, which decouples the status information from `oc adm upgrade` and provides specific information regarding a cluster update, including the status of the control plane and worker node updates.
```

Comment:
```suggestion
Previously, the `oc adm upgrade` command provided limited information about the status of a cluster update. This release adds the `oc adm upgrade status` command, which decouples status information from the `oc adm upgrade` command and provides specific information regarding a cluster update, including the status of the control plane and worker node updates.
```

WTYD? I attempted to minimize sentence length and keep the required `<command_name> command` language to a minimum.

Reply:
works for me :) great idea